www/source/blog/2018-06-13-base-plans-refresh.html.md

```diff
@@ -0,0 +1,170 @@
+
+## Base Plans 101
+
+Currently, there are about 500 Habitat Plans under the Core Origin on Builder.
+
+These include everything from services like Postgresql to binaries like Go to very low level system libraries like GCC, all of which you can easily pull into your own Habitat artifacts.
+
+### What's a Base Plan?
+
+The Base Plans are a group of Plans (mostly low level systems libraries) that:
+* are used by nearly every other Plan (or a dependency of that Plan) on Builder
+* are used to build Habitat itself
+* need to be built in a certain order.
+
+You can see [the full list of Base Plans](https://github.com/habitat-sh/Core-Plans/blob/master/CODEOWNERS) in the Core Plans CODEOWNERS file.
+
+Any time one of these Plans is updated (especially ones that are nearly universally used, like [glibc](https://en.wikipedia.org/wiki/GNU_C_Library) and [gcc](https://gcc.gnu.org/), all other Base Plans which depend on it need to be built in a certain order, and then all Plans that depend (or depend on a Plan that depends) on those Base Plans also need to be rebuilt. We call this the "rebuild the world" scenario.
+
+### Why are you refreshing them?
+
+Many of our current Base Plans are on very old versions (i.e. the current version of gcc on Builder is 5.2.0, when the most recent stable release of gcc is 8.1.0).  It is vital to update them - and consequently every Plan that depends on them - to more recent stable versions.
+
+Many of them also must be updated simultaneously - if we update gcc to 8.1 and it requires a later version of glibc than we current have, we must also update glibc.
+
+### What else does it affect?
+
+We are simultaneously refreshing nearly every [Core Plan](https://github.com/habitat-sh/Core-Plans/) at the same time as we are refreshing the Base Plans.  This is because the majority (if not all) of the Core Plans depend on at least one of the Base Plans in some fashion (or, at the very least, a Base Plan is in their dependency chain).
+
+## Base Plans and You
+
+So far we've discussed how and why we are refreshing the Base Plans and Core Plans, but what does that mean for the Plans in your own origin?
+
+### How does this affect me?
+
+Chances are, any Plan within your origin depends on at least one of the Core Base Plans somewhere in its chain of dependencies.  When that Plan is updated, your Plan also must be rebuilt.  This sounds simple, but if you have Plans in your origin that depend on other Plans - and those Plans depend on a Core Plan (let's say glibc), all of them must be built with the same version of glibc.  Let's look at this visually.
+
+### Dependencies
+
+Let's say I have a Plan under my origin (nellshamrell) called widget_world - let's say it lives at nellshamrell/widget_world on Builder.
+
+```
+nellshamrell/widget_world
+```
+
+Now, let's say that Plan depends on two more Plans nellshamrell/widget and nellshamrell/world (yes, it's contrived, just go with it for now).
```

Comment:
You might want a comma or a dash between `Plans` and `nellshamrell` to separate.